templates/pages/index.md.erb

```diff
@@ -36,8 +36,8 @@ I do my best, and I like it more than Russian, probably because of its laconism
 And I want to learn it more, and I think it's easier when living (or communicating)
 with English speakers.
 
-I use my devices (smartphone, PC, etc.) and all services (sites) in English,
-also I search information almost every time in English,
+I use my devices (smartphone, PC, etc.) and all services (sites) in English.
+Also, I search information almost all the time in English,
```

Comment:
It's more like "every attempt", not "while".

Reply:
I can agree on this one, yeah. Thanks for expanding my English knowledge boundaries